test/modules/main/bundle/deckgl.spec.js

```diff
@@ -19,10 +19,10 @@
 // THE SOFTWARE.
 
 import test from 'tape-catch';
-import deckgl from 'deck.gl/../bundle';
+import * as deckgl from 'deck.gl/../bundle';
```

Comment:
These imports look really weird? Assume the bundle is in the root directory? Can we place the bundle in the dist folder so we don't have to use `..`? If we want to avoid breaking, we can add a copy of the bundle there?